FirebaseAppCheck.podspec

```diff
@@ -36,7 +38,9 @@ Pod::Spec.new do |s|
   ]
   s.public_header_files = base_dir + 'Sources/Public/FirebaseAppCheck/*.h'
 
-  s.framework = 'DeviceCheck'
+  s.ios.framework = 'DeviceCheck'
+  s.osx.framework = 'DeviceCheck'
+  s.tvos.framework = 'DeviceCheck'
```

Comment:
I think we may need to do `weak_frameworks` if we're pushing back support for older versions that this framework isn't available

Reply:
Good point, let me update it.

Reply:
done